app/redux/reducers/SettingsReducer.js

```diff
@@ -22,10 +28,28 @@ const defaultState = {
     wasMetricEdited: false,
     showRemoved: false,
     isExportingCSV: false,
-    lastExportCSVDate: null,    
+    lastExportCSVDate: null,
+    metric1: 'Velocity',
+    quantifier1: 'Last Rep',
+    metric2: 'PKV',
+    quantifier2: 'Last Rep',
+    metric3: 'PKH',
+    quantifier3: 'Last Rep',
+    metric4: 'ROM',
+    quantifier4: 'Last Rep',
+    metric5: 'Duration',
+    quantifier5: 'Last Rep',
+    isEditingMetric: false,
+    isEditingAvgMetric: false,
+    isEditingBestEverMetric: false,
+    isEditingQuantifier: false,
+    currentMetricPosition: null,
+    currentQuantifierPosition: null
 };
 
```

Comment:
- [ ]  I've an issue with how we're treating non action constants. I let it go before, but my instinct is that the constants for metrics and quantifiers needs to be defined somewhere and imported here